src/libraries/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -53,32 +54,34 @@ public static partial class Http
 
             public static readonly Uri RemoteEchoServer = new Uri("http://" + Host + "/" + EchoHandler);
             public static readonly Uri SecureRemoteEchoServer = new Uri("https://" + SecureHost + "/" + EchoHandler);
-            public static readonly Uri Http2RemoteEchoServer = new Uri("https://" + Http2Host + "/" + EchoHandler);
-            public static readonly Uri[] EchoServerList = new Uri[] { RemoteEchoServer, SecureRemoteEchoServer, Http2RemoteEchoServer };
+            public static readonly Uri Http2RemoteEchoServer = new Uri("http://" + Http2Host + "/" + EchoHandler);
+            public static readonly Uri Http2SecureRemoteEchoServer = new Uri("https://" + Http2SecureHost + "/" + EchoHandler);
+            public static readonly Uri[] EchoServerList = new Uri[] { RemoteEchoServer, SecureRemoteEchoServer, Http2RemoteEchoServer, Http2SecureRemoteEchoServer };
 
             public static readonly Uri RemoteVerifyUploadServer = new Uri("http://" + Host + "/" + VerifyUploadHandler);
             public static readonly Uri SecureRemoteVerifyUploadServer = new Uri("https://" + SecureHost + "/" + VerifyUploadHandler);
-            public static readonly Uri Http2RemoteVerifyUploadServer = new Uri("https://" + Http2Host + "/" + VerifyUploadHandler);
+            public static readonly Uri Http2RemoteVerifyUploadServer = new Uri("https://" + Http2SecureHost + "/" + VerifyUploadHandler);
```

Comment:
Following the naming convention, shouldn't the name be `Http2SecureRemoteVerifyUploadServer` so its clear in that wherever the variable is used that it references `Http2SecureHost`
Same for L69 (`Http2SecureRemoteDeflateServer`), L70 (`Http2SecureRemoteGZipServer`), L76 `Http2SecureServers`).

I take it that all the tests that use these variables were intended to use the secure form, and so we no longer need the non-secure forms right?

Reply:
Your logic is correct, but it would propagate to too many places. I would keep it as is.